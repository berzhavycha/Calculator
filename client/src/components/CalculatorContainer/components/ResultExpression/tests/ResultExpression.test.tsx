import { render } from '@testing-library/react';
import { ResultExpression } from '../index';

describe('ResultExpression component', () => {
  test('matches the snapshot', () => {
    const mockProps = {
      result: '14',
      errorMessage: 'Invalid expression',
    };

    const { container } = render(
      <ResultExpression
        result={mockProps.result}
        errorMessage={mockProps.errorMessage}
      />
    );

    expect(container).toMatchSnapshot();
  });
});
