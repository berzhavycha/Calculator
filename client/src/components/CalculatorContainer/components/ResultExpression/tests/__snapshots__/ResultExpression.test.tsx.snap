// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ResultExpression component > matches the snapshot 1`] = `
<div>
  <div
    class="mb-4"
  >
    <div
      aria-label="result-expression"
      class="result m-2 text-3xl text-right"
    >
      4s+s
    </div>
    <div
      aria-label="error-expression"
      class="error-block text-red-500 h-6 text-right"
    />
  </div>
</div>
`;
